задание 1.py

```diff
@@ -0,0 +1,10 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+
+index_missing_item = 4
```

Comment:
В данном случае, чтобы сделать решение более универсальным, можно использовать метод поиска индекса искомого элемента
```suggestion
index_missing_item = numbers.index(None)
```